### PR TITLE
feature - user detail screen

### PR DESCRIPTION
**Pull request by at-duytran**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/ViewModel/UserProfile/UserProfileCellViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import Foundation
+import MVVM
+
+final class UserProfileCellViewModel: MVVM.ViewModel {
+    var iconImage: UIImage
+    var title: String
+
+    init(item: ProfileRowType) {
+        self.iconImage = item.item.iconImage
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/UserProfile/UserProfileCellViewModel.swift]: ? item.item

Comment (at-tuanmai2) [inline MyApp/ViewModel/UserProfile/UserProfileCellViewModel.swift]: ? self

File: MyApp/ViewModel/UserProfile/UserProfileCellViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        self.title = item.item.title
```
Comment (at-tuanmai2) [inline MyApp/ViewModel/UserProfile/UserProfileCellViewModel.swift]: nt

**Pull request merged**